xapian-core/bin/xapian-replicate-server.cc

```diff
@@ -26,7 +26,7 @@
 #include <xapian.h>
 
 #include "gnu_getopt.h"
-
+#include "parseint.h"
 #include <cstdlib>
 #include <iostream>
```

Comment:
Another unhelpful removal of a blank line between groups of headers (and more below).